### PR TITLE
FullPlanner- Initial App Check

### PR DESCRIPTION
**Pull request by Vignesh-Thangamariappan**

### CODE REVIEW

File: MyAssistant/Login.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+//
+//  Created by user on 12/4/17.
+//  Copyright © 2017 Vignesh. All rights reserved.
+//
+
+import UIKit
+import FacebookLogin
+import FBSDKLoginKit
+
+class Login: UIViewController, LoginButtonDelegate{
```
Comment (samsonsunny) [inline MyAssistant/Login.swift]: make it as LoginViewController

File: MyAssistant/Login.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    UserDefaults.standard.set(pic, forKey: "userImage")
+                    UserDefaults.standard.set(self.dict["name"], forKey: "userName")
+                    UserDefaults.standard.set(self.dict["email"],forKey:"userMail")
+                    self.goToTabBar()
+
+                }
+            }
+            )}
+    }
+    func goToTabBar() {
```
Comment (samsonsunny) [inline MyAssistant/Login.swift]: have a proper access specifier. If the method is not uses outside of the class scope make it as private

File: MyAssistant/MainNavigation.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+//
+//  Created by user on 12/4/17.
+//  Copyright © 2017 Vignesh. All rights reserved.
+//
+
+import UIKit
+import FacebookLogin
+import FBSDKLoginKit
+
+class MainNavigation: UINavigationController {
```
Comment (samsonsunny) [inline MyAssistant/MainNavigation.swift]: MainNavigationController

File: MyAssistant/Settings.swift [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+//
+//  Created by user on 12/4/17.
+//  Copyright © 2017 Vignesh. All rights reserved.
+//
+
+import UIKit
+import FacebookLogin
+import FBSDKLoginKit
+
+class Settings: UITableViewController {
```
Comment (samsonsunny) [inline MyAssistant/Settings.swift]: SettingsTableViewController

File: MyAssistant/Settings.swift [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    
+    let loginManager = LoginManager()
+    @IBOutlet weak var profileImage: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    let defaults = UserDefaults.standard
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if let user = defaults.value(forKey: "userName") as? String, let mail = defaults.value(forKey: "userMail") as? String, let image = defaults.value(forKey: "userImage") as? String {
```
Comment (samsonsunny) [inline MyAssistant/Settings.swift]: make it as guard statement

**Comment by Vignesh-Thangamariappan** [comment]
Changes have been made

**Pull request closed without merging**